data/searchers/map planets.txt

```diff
@@ -77,9 +77,10 @@ planet Salus
 	landscape land/clouds_02
 	description `Salus is a very cloudy world, notable for its heat and the red tint that the cloud cover gives its sun. The largest producer of medical products of all Searcher planets, it is very rich, and boasts the only major military base in all of Searcher space. Your ship is scanned upon landing, and you are carefully watched to make sure you don't attempt to do anything illegal.`
 	spaceport `The spaceport on Salus is very large, dominated by the military base. There is a lot of traffic here, but it has managed to avoid the worst of the congestion by the use of an extraordinarily complicated transportation network. The landing pads are generously sized, and there are enough if them to land even the very largest of fleets. As soon as you landed, you were provided with a guide instructing you in the operation of the transportation network.`
+	security 1
```

Comment:
I'm going to leave this out for now, but thank you for reminding me that I never did assign security or tribute data to any of these planets.

```suggestion
```

Reply:
It says that you can't do anything without being watched though.

Reply:
Yeah, I've added a "security" 0.9 on it in my last commit. I _might_ bump it up to 1, since it fits the description better, but I might also just keep it as-is since the only current factions with such high security are the Hai, Quarg and Pug. The Searchers can do a lot of scans, but they can't necessarily recognize alien illegal stuff for what it is.